bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-slick-carousel",
-  "version": "3.1.3",
+  "version": "3.1.4",
```

Comment:
New features should be a minor version bump, if following semver (which I'd always recommend)